cmd/kubeadm/app/cmd/token.go

```diff
@@ -208,8 +212,18 @@ func RunCreateToken(out io.Writer, client clientset.Interface, token string, tok
 		}
 	}
 
+	// validate groups
+	for _, group := range groups {
```

Comment:
Please add a TODO that as soon as https://github.com/kubernetes/client-go/issues/114 is fixed; we should get rid of this.
This should also be synced with what we do below and splitted out to its own func.